src/utils/formUtil/eachComponent.ts

```diff
@@ -43,7 +43,8 @@ export function eachComponent(
           eachComponent(column.components, fn, includeAll, compPaths, component),
         );
       } else if (info.hasRows) {
-        component.rows.forEach((row: any) => {
+        const comp = component.component || component;
```

Comment:
Can you explain this change? Is this `table` component specific? If so, it might be beneficial to have a `info.isTable` or something to make it explicitly clear that we have an edge case with the `table` component